layers/+lang/bibtex/packages.el

```diff
@@ -110,3 +112,48 @@
         "ld" 'doi-utils-add-bibtex-entry-from-doi
         "li" 'isbn-to-bibtex
         "lp" 'pubmed-insert-bibtex-from-pmid))))
+
+(defun bibtex/init-ebib ()
+  (use-package ebib
+    :defer t
+    :init
+    (spacemacs/set-leader-keys "ate" 'ebib)
+    :config
+    (setq ebib-bibtex-dialect 'biblatex)
+
+    (evilified-state-evilify-map ebib-index-mode-map
+      :mode ebib-index-mode
+      :bindings
+      "j" 'ebib-next-entry
+      "k" 'ebib-prev-entry
+      "J" 'evil-scroll-page-down
+      "K" 'evil-scroll-page-up
+
+      "/" 'ebib-search
+      "n" 'ebib-search-next)
+
+    (spacemacs/set-leader-keys-for-major-mode 'ebib-index-mode
+      "j" 'ebib-jump-to-entry
+      "k" 'ebib-kill-entry
+      "b" 'biblio-lookup)
+
+    (evilified-state-evilify-map ebib-entry-mode-map
+      :mode ebib-entry-mode)
+
+    (evilified-state-evilify-map ebib-log-mode-map
+      :mode ebib-log-mode)
+
+    (use-package ebib-biblio
```

Comment:
`ebib-biblio` is not a package. it's a feature of `ebib`.
So you should have `:after (biblio)` in `(use-package ebib)` instead.
Then in its `:config`, add `ebib-biblio-selection-import` key binding.

Reply:
`ebib-biblio` really has to get loaded separately (see [here](https://joostkremers.github.io/ebib/ebib-manual.html#integration-with-the-biblio-package)). But indeed I could use the `:after (biblio)` in `(use-package ebib)` and then `(require ebib-biblio)` in the :config. But I think it does not make much of a difference (or maybe does it?)

Reply:
I think the upstream author is abusing `use-package`.
For example, Spacemacs has defined a `core-configuration-layer` feature. You can try `(use-package core-configuration-layer)`, since it's found in load path, it will succeed.

Basically, any feature found in load path can be declared as a package via `use-package`. But it's really not a package.

Reply:
Your argument for using the appropriate 'labels' for the various cases, makes sense. But it seems that, practically, it should not make much of a difference as `use-package` for this case simply expands to `require` (see [here](https://www.reddit.com/r/emacs/comments/643dkt/use_package_vs_require_and_maybe_some_sorrowful/dfzya9d?utm_source=share&utm_medium=web2x&context=3)). But it is fine for me to change the 'terminology'.